test/jdk/java/awt/Mixing/AWT_Mixing/MixingPanelsResizing.java

```diff
@@ -157,25 +160,29 @@ public void run() {
             public void run() {
                 Point btnLoc = jbutton.getLocationOnScreen();
                 Color c = robot.getPixelColor(btnLoc.x + 5, btnLoc.y + 5);
-                if (!c.equals(jbColor)) {
+                System.out.println("Color picked for jbutton: "+c);
+                if (!isAlmostEqualColor(c, jbColor)) {
                     fail("JButton was not redrawn properly on AWT Panel during move");
                 }
 
                 btnLoc = awtButton.getLocationOnScreen();
                 c = robot.getPixelColor(btnLoc.x + 5, btnLoc.y + 5);
-                if (!c.equals(awtColor)) {
+                System.out.println("Color picked for awtButton: "+c);
+                if (!isAlmostEqualColor(c, awtColor)) {
                     fail("AWT Button was not redrawn properly on AWT Panel during move");
                 }
 
                 btnLoc = jbutton2.getLocationOnScreen();
                 c = robot.getPixelColor(btnLoc.x + 5, btnLoc.y + 5);
-                if (!c.equals(jb2Color)) {
+                System.out.println("Color picked for jbutton2: "+c);
```

Comment:
```suggestion
                System.out.println("Color picked for jbutton2: " + c);
```
Spaces around binary operators.
This applies to other similar cases.

Reply:
I have gone through the test and added spaces around binary operators.